Dockerfiles/nginx/default

```diff
@@ -0,0 +1,6 @@
+server {
+    listen 80;
```

Comment:
I would just include a note in the documentation that production deployments are expected to setup the necessary TLS stuff, in this container or elsewhere. That's really the (main) reason we advise to put nginx in front of the backend and the proxy.

Reply:
that Nginx shouldn't be there anyway, same reasoning as above: let's leave it like this for the moment and kick it later, when the problem is solved properly (i.e. CORS handled correctly)

I opened a ticket here: https://github.com/dedis/d-voting/issues/307